ui/daml/UI/Onboarding.daml

```diff
@@ -53,45 +47,37 @@ handleOnboardingInstruction operator customer mp ins = case ins of
     o@OnboardClearinghouse{..} -> do
       optClearingRole <- lookupByKey @Clearinghouse.Role (operator, customer)
       when (isNone optClearingRole) $ do
+        let account = Custody.createAccount custodian customer
         custodyCid <- createOrLookup Custody.Service with provider = custodian; ..
 
-        ccpAccount <- case optClearingAccount of
-          (Some an) -> return $ fromSomeNote "account lookup" $ Map.lookup (OnboardingAccount custodian an) mp
-          None      -> createAccount custodyCid operator custodian customer "Clearinghouse" []
-
         offerCid <- exerciseByKey @Operator.Role operator Operator.OfferClearingRole with provider = customer
-        void $ exercise offerCid Clearinghouse.Accept with ..
+        void $ exercise offerCid Clearinghouse.Accept with ccpAccount = account
 
     o@OnboardCustody{..} -> do
       offerCid <- exerciseByKey @Custodian.Role (operator, provider) Custodian.OfferCustodyService with ..
       void $ exercise offerCid Custody.Accept
 
     ot@OnboardTrading{..} -> do
+        let account = Custody.createAccount custodian customer
         custodyCid <- createOrLookup Custody.Service with provider = custodian; ..
```

Comment:
Is creating the `Custody.Service` here still technically necessary? From what I understand with this PR is that while you will need a custody service to trade, there's not any one custody service tied to the trading service. So I can see it make sense either way - require a custodian in the instruction and create it here to ensure the customer has at least one custody service, or skip creating the custodian to avoid the confusion of the custody service not being tied to the trading service.

Reply:
Agree, I think this was needed to get the necessary accounts for the trading service, but that's not required anymore. So I *think" it should be safe to remove, but check if there's some implicit dependency on this being created here

Reply:
I actually had the same thought but I left it in. It does look strange in the quick setup as for some roles you have to specify the `Custody` service provider multiple times. 

I do think we should review this but perhaps outside of this PR, if you agree ?